src/udev/udev-builtin-net_setup_link.c

```diff
@@ -21,6 +22,29 @@ static int builtin_net_setup_link(UdevEvent *event, int argc, char **argv, bool
         if (argc > 1)
                 return log_device_error_errno(dev, SYNTHETIC_ERRNO(EINVAL), "This program takes no arguments.");
 
+        sd_device_action_t action;
+        r = sd_device_get_action(dev, &action);
+        if (r < 0)
+                return log_device_error_errno(dev, r, "Failed to get action: %m");
+
+        if (!IN_SET(action, SD_DEVICE_ADD, SD_DEVICE_BIND, SD_DEVICE_MOVE)) {
+                log_device_debug(dev, "Skipping to apply .link settings on '%s' uevent.",
+                                 device_action_to_string(action));
+
+                /* Import previously assigned .link file name. */
+                const char *value;
+                if (sd_device_get_property_value(event->dev_db_clone, "ID_NET_LINK_FILE", &value) >= 0)
+                        (void) udev_builtin_add_property(dev, test, "ID_NET_LINK_FILE", value);
+                if (sd_device_get_property_value(event->dev_db_clone, "ID_NET_LINK_FILE_DROPINS", &value) >= 0)
+                        (void) udev_builtin_add_property(dev, test, "ID_NET_LINK_FILE_DROPINS", value);
```

Comment:
don#t we have a helper for that? i.e. udev_builtin_copy_property()? or should we add one?